src/node/utils/isNodeExceptionLike.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Determines if the given value is shaped like a Node.js exception.
+ * Node.js exceptions have additional information, like
+ * the `code` and `errno` properties.
+ *
+ * In some environments, particularly jsdom/jest these may not
+ * instances of `Error` or its subclasses, despite being similar
+ * to them.
+ */
+export function isNodeExceptionLike(
```

Comment:
renamed this function and added some docs that it's _very similar_ to a `NodeException` but isn't guaranteed to be